### PR TITLE
[Merged by Bors] - chore(shell/lean): remove lean --doc.

### PR DESCRIPTION
**Pull request by Julian**
It's broken. Use doc-gen.

Closes: #465

### CODE REVIEW

File: src/shell/lean.cpp [hunk 1]
```diff
@@ -491,9 +489,6 @@ int main(int argc, char ** argv) {
         case 'n':
             native_output         = optarg;
             break;
-        case 'r':
-            doc = optarg;
```
Comment (gebner) [inline src/shell/lean.cpp:495]: Could you also remove the `doc` variable and the `leandoc.cpp` file, please?

Comment (Julian) [reply src/shell/lean.cpp:495]: Whoops, knew I'd miss something... Fixed.

**Comment by gebner** [comment]
Thanks!
bors r+

**Comment by Julian** [comment]
(And now removed from the Makefile). Does that CI thing listen to scrubs or will it ignore me

bors r+

**Pull request closed without merging**